src/main/java/eu/xenit/gradle/docker/alfresco/DockerAlfrescoPlugin.java

```diff
@@ -176,6 +171,11 @@ private static void configureAddWarDockerFile(Project project, String name, Task
                             .from(project.getConfigurations().named(warName.toLowerCase() + "Amp"));
                 }));
 
+        tasks.add(project.getTasks()
```

Comment:
Does changing the order do something? Or is just for esthetics?

Reply:
The order in the `tasks` list is important, as the outputs of these tasks are merged together in order.

So, later entries in the `tasks` list will be merged on top of earlier entries in this list.

For most operations, this order is not important, since they all work on a 'stripped' Alfresco war, so there are no conflicts between files. But the `log4j.properties` file is one of those files that is part of the stripped war.

So, modifying this file has to happen last. Else, the other stripped wars that are merged together will overwrite the changes made.